### ISSUE TITLE
Can't update after running your script

### ISSUE THREAD

**Issue by 888iee** [Nyr/wireguard-install#49]
Hi,
I freshly installed Pi-Os on rpi0, did 
`sudo apt update && upgrade -y`
then I downloaded your script and executed it with
`sudo bash wireguard-install.sh`
VPN runs fine but now when I execute apt update it shows the following.
I don't know if this is a pi special thing or it happens in general.

> Hit:1 http://raspbian.raspberrypi.org/raspbian buster InRelease
> Get:2 http://deb.debian.org/debian buster-backports InRelease [46.7 kB]
> Hit:3 http://archive.raspberrypi.org/debian buster InRelease
> Get:4 https://download.docker.com/linux/raspbian buster InRelease [33.6 kB]
> Err:2 http://deb.debian.org/debian buster-backports InRelease
>   The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 04EE7237B7D453EC NO_PUBKEY 648ACFD622F3D138
> Reading package lists... Done
> W: GPG error: http://deb.debian.org/debian buster-backports InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 04EE7237B7D453EC NO_PUBKEY 648ACFD622F3D138
> E: The repository 'http://deb.debian.org/debian buster-backports InRelease' is not signed.
> N: Updating from such a repository can't be done securely, and is therefore disabled by default.
> N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by Nyr** [comment]
I know I'm late, but the solution would've been:

```
apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC
apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 648ACFD622F3D138
```

**Comment by 888iee** [comment]
Sorry didn't try out till now. I don' think this fixes anything.
Now I get the following
> Get:1 http://deb.debian.org/debian buster-backports InRelease [46.7 kB]
> Get:2 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]
> Get:3 http://archive.raspberrypi.org/debian buster InRelease [32.6 kB]
> Get:4 https://download.docker.com/linux/raspbian buster InRelease [33.6 kB]
> Get:5 http://raspbian.raspberrypi.org/raspbian buster/main armhf Packages [13.0 MB]
> Err:1 http://deb.debian.org/debian buster-backports InRelease
>   The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 648ACFD622F3D138 NO_PUBKEY 0E98404D386FA1D9
> Get:6 http://archive.raspberrypi.org/debian buster/main armhf Packages [393 kB]
> Get:7 http://raspbian.raspberrypi.org/raspbian buster/contrib armhf Packages [58.8 kB]
> Get:8 https://download.docker.com/linux/raspbian buster/stable armhf Packages [19.1 kB]
> Get:9 http://raspbian.raspberrypi.org/raspbian buster/non-free armhf Packages [104 kB]
> Reading package lists... Done
> W: GPG error: http://deb.debian.org/debian buster-backports InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 648ACFD622F3D138 NO_PUBKEY 0E98404D386FA1D9
> E: The repository 'http://deb.debian.org/debian buster-backports InRelease' is not signed.
> N: Updating from such a repository can't be done securely, and is therefore disabled by default.
> N: See apt-secure(8) manpage for repository creation and user configuration details.
> N: Repository 'http://archive.raspberrypi.org/debian buster InRelease' changed its 'Suite' value from 'testing' to 'oldstable'

**Comment by Nyr** [comment]
`apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 648ACFD622F3D138`
`apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0E98404D386FA1D9`

If this does not fix your error, please provide the output of the above commands.

Anyway now that Debian 11 is available I suggest to just use that, as it no longer requires backported packages.
